src/Apps/Artwork/Components/PrivateArtwork/PrivateArtworkAboutArtist.tsx

```diff
@@ -97,9 +98,15 @@ export const PrivateArtworkAboutArtist: React.FC<PrivateArtworkAboutArtistProps>
                 </Box>
 
                 <Box>
-                  <Text variant="lg-display" color="white100" fontWeight="bold">
-                    {artist.name}
-                  </Text>
+                  <RouterLink to={artist.href} display="block">
+                    <Text
+                      variant="lg-display"
+                      color="white100"
+                      fontWeight="bold"
+                    >
+                      {artist.name}
+                    </Text>
+                  </RouterLink>
```

Comment:
Clicking on the artist name here will take the user to the artist page
<img width="1184" alt="Screenshot 2024-10-09 at 9 22 17 AM" src="https://github.com/user-attachments/assets/bd8a9883-f2e8-491f-be9c-8613ecc79cf9">


http://localhost:4000/artist/lucy-dodd